Core/clim-basic/geometry/transforms.lisp

```diff
@@ -702,7 +702,7 @@ real numbers, and default to 0."
 (defun atan* (x y)
   ;; atan like we need it.
   ;; For us, phi=0 along the x axis and angles are always between 0 and 2pi.
-  (let ((r (atan y x)))
+  (let ((r (atan (float y pi) x)))
```

Comment:
Why is this needed?

Reply:
because we compare the result with pi (i.e in correct-angle and angle-contains-angle-p); I recall I had a very tangible problem with that but I don't remember what was it now.

Reply:
Please add a FIXME to either remove the `float` call or document why it is needed, then.

Reply:
commented - please see last commit

(and for amusement: https://irclog.tymoon.eu/freenode/%23clim?around=1574156524#1574156524)

Reply:
done (please see last commit)